packages/validation/docs/valibot-api-index.md

```
@@ -0,0 +1,1722 @@
+# Valibot API Index
+
+_Scraped from [valibot.dev/api](https://valibot.dev/api/) — 2025-07-30T20:07:20.658Z_
+
+> This file is auto-generated. Edit the script to update.
+
+## Schemas
+
+- [any](https://valibot.dev/api/any/): Creates an any schema.
+  ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)